### ISSUE TITLE
Fix epoch runs on testnet

### ISSUE THREAD

**Issue by devey** [rysk-finance/dynamic-hedging#320]
breaks if you try to run the rpm command. Need to double check as this is something from weeks ago but I was not able to move epoch (might had to do with private keys and accesses)

**Comment by MeanBoyCousin** [comment]
Looks like the scripts and JSON imports are still referencing Rinkeby instead of Goerli. Possible cause?

For this, are the epochs supposed to be run routinely like prod or do we just run them whenever we need to test interactions with the DHV? If the former, would it be worth running this epoch command for test-net on a workflow using cron? We could have it automatically execute the epoch every week at a fixed time this way. If the later, probably just need to fix the command.

**Comment by devey** [comment]
> Looks like the scripts and JSON imports are still referencing Rinkeby instead of Goerli. Possible cause?
> 
> For this, are the epochs supposed to be run routinely like prod or do we just run them whenever we need to test interactions with the DHV? If the former, would it be worth running this epoch command for test-net on a workflow using cron? We could have it automatically execute the epoch every week at a fixed time this way. If the later, probably just need to fix the command.

hey @MeanBoyCousin, many front-end functionalities can be only tested by running epochs (deposits, withdraws, charts, not sure about the OTC options) but yeah it's more something we need in local development for being able to test and iterate on front-end; on mainnet we run these epochs on a sort of weekly basis manually (it also requires to have some parameters ready to input) so I would say we don't need the cronjob for now but getting the commands to work would be a good start, although we better wait until @kjr217 is done with the deployment scripts to make sure there is no blockers :)

**Comment by MeanBoyCousin** [comment]
So we can run the epochs for local testing whenever we want to test functionality? I assume this is how we test code consumed by the `useContract` hook?

**Comment by devey** [comment]
> So we can run the epochs for local testing whenever we want to test functionality? I assume this is how we test code consumed by the `useContract` hook?

I've still not run it once as it was not working, but yes the idea is that we fix the scripts and document the process for testing things in a markdown in the repo, including running epochs locally to test much of the smart contracts functionalities

**Comment by MeanBoyCousin** [comment]
@devey Is this sorted now we have deployed testnet?

**Comment by devey** [comment]
@MeanBoyCousin yes, I'll work on it soon (for now I've been doing it manually by following the steps in `CreateOptionProducts`):

```ts
it("pauses trading and executes epoch", async () => {
	await liquidityPool.pauseTradingAndRequest()
	await portfolioValuesFeed.fulfill(weth.address, usd.address)
	await liquidityPool.executeEpochCalculation()
	await liquidityPool.redeem(toWei("10000000"))
})
```
